index.php

```diff
@@ -359,15 +373,16 @@
 	$rows .
 '</table>';
 
-?>
-<script src="js/DetailsFieldLayout.js"></script>
-<script src="js/PatchSelectWidget.js"></script>
-<script src="js/index.js"></script>
-<?php
 echo '<script>
+window.pd = window.pd || {};
 pd.wikiPatches = ' . json_encode( $wikiPatches ) . ';
 pd.config = ' . json_encode( [
 	'phabricatorUrl' => $config['phabricatorUrl']
 ] ) . ';
 </script>';
```

Comment:
Needed to change the load order as we can set PatchSelectWidget value in the constructor if values loaded from query string. wikiPatches needs to be available when the PSW changes value.